core/scripts/chaincli/handler/keeper.go

```diff
@@ -193,8 +217,31 @@ func (k *Keeper) approveFunds(ctx context.Context, registryAddr common.Address)
 	log.Println(registryAddr.Hex(), ": KeeperRegistry approved - ", helpers.ExplorerLink(k.cfg.ChainID, approveRegistryTx.Hash()))
 }
 
+func (k *Keeper) verifyRegistry21(address common.Address, params ...string) {
+	// Change directory to the contracts directory where the hardhat.config.js file is located
+	k.changeDirectory("../../../contracts")
+
+	// Append the address and params to the commandArgs slice
+	commandArgs := []string{address.String()}
+	commandArgs = append(commandArgs, params...)
+
+	// Format the command string with the commandArgs
+	command := fmt.Sprintf(
+		"NODE_HTTP_URL='%s' ETHERSCAN_API_KEY='%s' pnpm hardhat verify --network network %s",
+		k.cfg.NodeHttpURL,
+		k.cfg.EtherscanAPIKey,
+		strings.Join(commandArgs, " "),
+	)
+
+	log.Println(command)
+	err := k.runCommand(command)
```

Comment:
You can collapse this line and the following if into:

`if err := k.runCommand(command); err != nil {...}`